### ISSUE TITLE
Feature\Docs Request: oscdump and oscsend

### ISSUE THREAD

**Issue by CastixGitHub** [hannesbraun/ayaya#1]
liblo provides `oscdump` and `oscsend` and `oscsendfile`

https://fukuchi.org/works/oscsend/index.html.en

It also says that them are incompatible with the ones from Matt Wright
https://web.archive.org/web/20170823074534/http://cnmat.org/OpenSoundControl/dumpOSC.html
https://web.archive.org/web/20170823073917/http://cnmat.org/OpenSoundControl/clients/sendOSC.html

I recorded [some osc messages](https://web.archive.org/web/20170823073917/http://cnmat.org/OpenSoundControl/clients/sendOSC.html) with oscdump
but, since the messages are timestamped, I couldn't find an option in oscsend to reset them at the time of playing.

Having equivalent commands in ayaya would be great
Can you also provide some example usage in the readme? thanks!

**Comment by hannesbraun** [comment]
I'm probably still rather busy in the following days. After that, I may find the time to add some example usage and add your requested feature.

If I understood you correctly, you want to record and replay OSC messages, right?

**Comment by CastixGitHub** [comment]
> If I understood you correctly, you want to record and replay OSC messages, right?

Yes. I plan to collaborate with other people by thending them the osc recordings, so they should be able to play the messages in the future, this means oscsend needs to recalculate the timestamp of every message before it starts.

I'd like to keep compatibility with the format used by the liblo implementation

**Comment by hannesbraun** [comment]
I totally forgot about this, I'm very sorry. But just for the record, I'll likely won't implement this anytime soon either as it's simply not a priority for me. I've been using this tool every now and then and it already has all the features I need. But if you or someone else wants to implement a dump/send feature with liblo compatibility, I don't have any problem with that. I even started with the GUI part of this feature in the `dump-and-send` branch but I never finished it.

Also, I will archive the project on GitHub soon. The new repository location is on SourceHut: https://git.sr.ht/~hannes/ayaya
